test/src/XcooBee/Store/CachedDataTest.php

```diff
@@ -38,4 +50,11 @@ public function testClearStore()
         $this->assertNull(Store::getInstance()->getStore('key'));
         $this->assertNull(Store::getInstance()->getStore('key2'));
     }
+
+    public function testClearStore_StoreIsNull()
+    {
+        $store = new Store();
+        $this->assertEquals(null, $store->getStore('key', 'value'));
+        $this->assertEquals(null, $store->getStore('key'));
```

Comment:
We have calls to `getStore` but function named `testClearStore_...`. Shouldn't we be making a call to `clearStore`? I'm confused.